ember-cli-build.js

```diff
@@ -1,8 +1,8 @@
 /* eslint-env node */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
+const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
-  var app = new EmberAddon(defaults, {
+  const app = new EmberAddon(defaults, {
```

Comment:
> At the very least, if you feel strongly about making this change at this moment in this repo then you need to open issues in all the other repos to address the same so the difference between the repos, and the desire to reconcile them, is captured.

The only thing I feel strongly about at this moment is being able to use our already published lint rules in my app. If I have to go make N issues on N repositories I may or may not ever have any need of using, simply to be able to use our lint rules in my app, then I think our system is broken and does not encourage anyone to conform to the standardization you're trying to enforce.